docs/index.rst

```diff
@@ -49,6 +49,31 @@ frontend, like ``builddir``, to specify the build directory to use/re-use. You
 can find more information about them in the `build options page`_.
 
 
+Editable installs
+-----------------
+
+Editable installs allow you to install the project in such a way where you can
+edit the project source and have those changes be reflected in the installed
+module.
+
+The way it works on ``meson-python`` specifically, is that when you import a
+module of the project, it will be rebuilt. This means that project imports will
+take more time than usual.
+
+It might be helpful to see the output of the Meson_ commands. That can be
+achieved by using the ``editable-verbose`` config setting when installing the
+project, or afterwards, by setting the ``MESONPY_EDITABLE_VERBOSE`` environment
```

Comment:
A few thoughts:
- for `editable-verbose`, can you please add the exact thing to add to the command line? this isn't clear
- should we mark `MESONPY_EDITABLE_VERBOSE` as provisional (subject to change)?
- it needs to be specified what happens when both the cli flag and the env var are given for the initial build

Reply:
Also, I am already seeing that we need verbosity levels. Rebuilding 2 files followed by 1000+ lines of _"Installing xxx"_ is not an optimal user experience.

Reply:
> for `editable-verbose`, can you please add the exact thing to add to the command line? this isn't clear

> it needs to be specified what happens when both the cli flag and the env var are given for the initial build

Done.

> it needs to be specified what happens when both the cli flag and the env var are given for the initial build

Sure.

> Also, I am already seeing that we need verbosity levels. Rebuilding 2 files followed by 1000+ lines of _"Installing xxx"_ is not an optimal user experience.

This should be fixed by adding the missing `--only-changed` argument.